<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QMainWindow" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自动阅卷系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-1</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAutomaticGradingSystem">
    <property name="title">
     <string>阅卷</string>
    </property>
    <addaction name="actionInputAnswer"/>
    <addaction name="actionInputReply"/>
    <addaction name="actionGradingPaper"/>
    <addaction name="actionOutputExcel"/>
   </widget>
   <widget class="QMenu" name="menuBlurImageProcessing">
    <property name="title">
     <string>模糊图像复原</string>
    </property>
    <addaction name="action1_11"/>
    <addaction name="action1_12"/>
    <addaction name="action1_13"/>
    <addaction name="action1_14"/>
    <addaction name="action1_15"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionDisplayPicture"/>
    <addaction name="action2_2"/>
    <addaction name="action3_2"/>
    <addaction name="action4_2"/>
    <addaction name="action5_2"/>
    <addaction name="action6_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>输出</string>
    </property>
    <addaction name="action1_3"/>
    <addaction name="action2_3"/>
    <addaction name="action3_3"/>
    <addaction name="action4_3"/>
    <addaction name="action5_3"/>
    <addaction name="action6_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="action1_4"/>
    <addaction name="action2_4"/>
    <addaction name="action3_4"/>
    <addaction name="action4_4"/>
    <addaction name="action5_4"/>
    <addaction name="action6_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>阅卷</string>
    </property>
    <addaction name="action1_5"/>
    <addaction name="action2_5"/>
    <addaction name="action3_5"/>
    <addaction name="action4_5"/>
    <addaction name="action5_5"/>
    <addaction name="action6_5"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionBasicParameterSettings"/>
    <addaction name="actionPreprocessingPapers"/>
    <addaction name="actionExamPaperSetting"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAutomaticGradingSystem"/>
   <addaction name="menuTools"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menuBlurImageProcessing"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInputAnswer">
   <property name="text">
    <string>读入答案</string>
   </property>
  </action>
  <action name="actionInputReply">
   <property name="text">
    <string>读入试卷</string>
   </property>
  </action>
  <action name="actionGradingPaper">
   <property name="text">
    <string>开始阅卷</string>
   </property>
  </action>
  <action name="actionOutputExcel">
   <property name="text">
    <string>导出为Excel</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionDisplayPicture">
   <property name="text">
    <string>显示图片</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6_2">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_3">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5_3">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6_3">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_4">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_4">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5_4">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6_4">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_5">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_5">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5_5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6_5">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionBasicParameterSettings">
   <property name="text">
    <string>基本参数设置</string>
   </property>
  </action>
  <action name="action1_7">
   <property name="text">
    <string>试卷分值设置</string>
   </property>
  </action>
  <action name="action1_8">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action1_9">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action1_10">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_11">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_12">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_13">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_14">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_15">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionPreprocessingPapers">
   <property name="text">
    <string>预处理试卷</string>
   </property>
  </action>
  <action name="actionExamPaperSetting">
   <property name="text">
    <string>试卷设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
